core/arch/arm/plat-stm32mp1/main.c

```diff
@@ -98,6 +99,32 @@ void main_secondary_init_gic(void)
 	stm32mp_register_online_cpu();
 }
 
+#ifndef CFG_EMBED_DTB
+static TEE_Result init_stm32mp1_drivers(void)
+{
+	/* Without secure DTB support, some drivers must be inited */
+#ifdef CFG_STM32_ETZPC
+	stm32_etzpc_init(ETZPC_BASE);
+#endif
+
+	return TEE_SUCCESS;
+}
+driver_init(init_stm32mp1_drivers);
+#endif /*!CFG_EMBED_DTB*/
```

Comment:
It would be preferable to avoid/minimize the `#ifdefs` in `.c` files (yeah, I know we do have quite a lot of them already...). `stm32_etzpc_init()` could be defined as `static inline void stm32_etzpc_init(paddr_t addr __unused) {}` in the header file when `!defined(CFG_STM32_ETZPC)`.

Reply:
ok